packages/xo-web/src/common/custom-fields.js

```diff
@@ -14,11 +15,15 @@ import { connectStore } from './utils'
 import { Container, Col } from './grid'
 import { createGetObject } from './selectors'
 import { form } from './modal'
-import { Text } from './editable'
+import { Toggle } from './form'
 
 const CUSTOM_FIELDS_KEY_PREFIX = 'XenCenter.CustomFields.'
+const DATE_FORMAT = 'YYYY-MM-DD'
+const TIME_FORMAT = 'HH:mm:ss'
 
-const AddCustomFieldModal = decorate([
+const isInvalidDate = value => new Date(value).toString() === 'Invalid Date'
```

Comment:
See https://momentjs.com/guides/#/warnings/js-date/
 I used this function because I couldn't use `isValid` function if `value` is a string.

Reply:
Can't you use an explicit format? Also, this will accept formats that are not considered as dates by XAPI.